src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -481,7 +481,8 @@ private void getMicroserviceDetails(RoutingContext event) {
                                .put("maxTileLength", maxTileLength)
                                .put("maxPlaneWidth", maxPlaneWidth)
                                .put("maxPlaneHeight", maxPlaneHeight)
-                               .put("maxActiveChannels", MAX_ACTIVE_CHANNELS));
+                               .put("maxActiveChannels", MAX_ACTIVE_CHANNELS))
+                .put("supportMissingChannels", true);
```

Comment:
Since this is just a flag, I suggest to add this to the `features` array above and call it e.g. `partial-channels` to match the capitalization of the other features.